### ISSUE TITLE
Expected positive CVE id for jackson-core:2.8.3, but not showing in report

### ISSUE THREAD

**Issue by mrrobbins** [dependency-check/DependencyCheck#1176]
Expected positive identification of CVE-2016-7051 from

Dependency: 
```xml
<dependency>
  <groupId>com.fasterxml.jackson.core</groupId>
  <artifactId>jackson-core</artifactId>
  <version>2.8.3</version>
</dependency>
```

Maven output: https://gist.github.com/mrrobbins/a5a4f9af9f465951548a22663a08ee91
Demo: https://github.com/mrrobbins/vuln-dependency-checker/blob/master/pom.xml

I'm evaluating `dependency-check` and I tested two dependencies with known CVEs. The report identifies the CVE for `okhttp` but not `jackson-core`. But in the report, the CPE [link](https://nvd.nist.gov/view/vuln/search-results?adv_search=true&cves=on&cpe_version=cpe%3A%2Fa%3Afasterxml%3Ajackson%3A2.8.3) for Jackson takes you to the NIST page showing the vuln.

**Comment by aikebah** [comment]
Why would you expect this CVE to appear?
As can be read on the CVE summary it is an issue of jackson-dataformat-xml, which did not get a separate cpe entry from NIST, but it is a library separate of jackson-core.
So your dependency has no known vulnerabilities.

Would you add dependency
```xml
        <dependency>
            <groupId>com.fasterxml.jackson.dataformat</groupId>
            <artifactId>jackson-dataformat-xml</artifactId>
            <version>2.8.3</version>
        </dependency>
```

You would see this CVE alongside many others:

jackson-dataformat-xml-2.8.3.jar (cpe:/a:fasterxml:jackson-databind:2.8.3, cpe:/a:fasterxml:jackson:2.8.3, com.fasterxml.jackson.dataformat:jackson-dataformat-xml:2.8.3) : CVE-2017-15095, CVE-2017-17485, CVE-2017-7525, CVE-2016-7051, CVE-2018-7489, CVE-2018-5968

**Comment by mrrobbins** [comment]
Thanks for responding aikebah.

I expected to see a vuln in the report for this CPE because the CPE's link (in the report) takes you to a NIST page for Jackson which lists a High and Medium vuln.

Link: https://nvd.nist.gov/view/vuln/search-results?adv_search=true&cves=on&cpe_version=cpe%3A%2Fa%3Afasterxml%3Ajackson%3A2.8.3

Per your suggestion, I did add jackson-databind to my POM and a vulnerability was included in the report, as expected.

I think the disconnect for me here was jackson-core:2.8.3 is clean, but one of it's child components (databind) is not, and the NIST search page is pulling in results for all jackson*:2.8.3. Looking closer, the NIST results call out jackson-databind in the description, but not the explicit GAV, so a little confusing.

tl:dr NIST searches linked in the report are broader than the specific CPE listed in the report.

This issue can be closed.
